src/bun.js/bindings/BunObject.cpp

```diff
@@ -362,9 +363,11 @@ JSC_DEFINE_HOST_FUNCTION(functionBunSleep,
         return JSC::JSValue::encode(JSC::JSValue {});
     }
 
+    countdown = std::ceil(millisecondsValue.asNumber());
```

Comment:
cant it just be `int countdown =` instead of the declaration and the assignment happening on separate lines?

Reply:
Absolutely. This isn't Linux, after all :-)